### PR TITLE
[Teams Scenario] Notification only bot

### PR DESCRIPTION
**Pull request by Virtual-Josh**

### CODE REVIEW

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.Luis/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.QnA/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.ApplicationInsights/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.Azure/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.Dialogs/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.TemplateManager/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Configuration/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Connector/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Schema/4.3.1)

File: tests/Teams/NotificationOnly/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            }
+            else
+            {
+                app.UseHsts();
+            }
+
+            app.UseDefaultFiles();
+            app.UseStaticFiles();
+
+            //app.UseHttpsRedirection();
```
Comment (stevengum) [inline tests/Teams/NotificationOnly/Startup.cs]: @johnataylor what are the situations where this line would be uncommented? I see it occasionally and it's always commented out... 😞

File: tests/Teams/NotificationOnly/NotificationOnly.csproj [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\libraries\integration\Microsoft.Bot.Builder.Integration.AspNet.Core\Microsoft.Bot.Builder.Integration.AspNet.Core.csproj" />
+    <ProjectReference Include="..\..\..\libraries\Microsoft.Bot.Builder.Teams\Microsoft.Bot.Builder.Teams.csproj" />
+    <ProjectReference Include="..\..\..\libraries\Microsoft.Bot.Schema.Teams\Microsoft.Bot.Schema.Teams.csproj" />
```
Comment (stevengum) [inline tests/Teams/NotificationOnly/NotificationOnly.csproj]: I think this reference can be removed 😄

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.Luis/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.QnA/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.ApplicationInsights/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.Azure/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.Dialogs/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Builder.TemplateManager/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Configuration/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Connector/4.3.1)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll** compared against [version 4.3.1](https://www.nuget.org/packages/Microsoft.Bot.Schema/4.3.1)

**Pull request merged**